virtual_assistant/.gitignore

```diff
@@ -0,0 +1,165 @@
+BotClass
```

Comment:
please move this file to the project's root
do we need all these lines?

Reply:
No, we dont'n. This line could be deleted